src/main/resources/i18n/Messages_en.properties

```diff
@@ -167,6 +167,28 @@ notification.observation.upcoming.email.body.text.3=To complete observations, do
 notification.observation.upcoming.email.subject=Upcoming Observation
 notification.organization.email.buttonIntro=Please click the button below to go to your Terraware account where you can view the organization.
 notification.organization.email.linkIntro=Please click the link below to go to your Terraware account where you can view the organization.
+# {0} is the scientific name of a species. {1} is the project name it was added to.
+notification.participantProjectSpecies.added.app.body={0} has been submitted for use for {1}.
+# {0} is the participant name.
+notification.participantProjectSpecies.added.app.title=A species has been added to a project for {0}.
+# {0} is the scientific name of a species.
+notification.participantProjectSpecies.edited.app.body={0} has been edited and is ready for approval.
+# {0} is the participant name.
+notification.participantProjectSpecies.edited.app.title=An approved species has been edited for {0}.
```

Comment:
Just noting that Phrase will move these down to be in alphabetical order when it publishes its PR to add translations.